config/bidderinfo.go

```diff
@@ -206,6 +209,11 @@ type InfoReaderFromDisk struct {
 	Path string
 }
 
+const (
+	ResponseFormatIFrame   = "b" // b = blank HTML response
+	ResponseFormatRedirect = "i" // i = image response
```

Comment:
Nitpick: Since these are moved out of the sync package, consider renaming to `SyncResponseFormatIFrame` and `SyncResponseFormatRedirect` to add back that context.